go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/gorilla/mux v1.7.4
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/mattn/go-isatty v0.0.12
+	github.com/otiai10/copy v1.1.1
```

Comment:
is this dep required?

Reply:
I use it to copy `testdata` directories into testcase-specific directories, so that changes don't propogate